modules/api/go.mod

```
@@ -57,4 +57,5 @@ require (
 )
 
 replace github.com/kubeedge/dashboard/client => ../common/client
+
```

```suggestion
replace (
           github.com/kubeedge/dashboard/client => ../common/client
           github.com/kubeedge/dashboard/errors => ../common/errors
)
```

You can combine the two replaces